planner-agent/src/workflows/vibeTodoCreator/utils.py

```
@@ -0,0 +1,243 @@
+    IN_REVIEW = "in_review"
+    DONE = "done"
+    FAILED = "failed"
+
+
+class SwarmBountyType(str, Enum):
+    DOCUMENT_SUMMARIZER = "document-summarizer"
+    FIND_BUGS = "find-bugs"
+    BUILD_FEATURE = "build-feature"
+
+
+class TaskAssignedInfo(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)